src/util/ReactBatchedUpdates.native.js

```diff
@@ -0,0 +1,2 @@
+const {unstable_batchedUpdates} = require('ReactNative');
```

Comment:
File needs standard copyright headers.  Rename to `Recoil_ReactBatchedUpdates.native.js`.